sdk/monitor/azure-monitor-query/azure/monitor/query/_models.py

```diff
@@ -161,60 +162,30 @@ class LogsQueryRequest(InternalLogQueryRequest):
      with either end_time or duration.
     :keyword datetime end_time: The end time till which to query the data. This should be accompanied
      with either start_time or duration.
-    :param workspace: Workspace Id to be included in the query.
```

Comment:
You want to keep it as workspace_id, right?

Reply:
no- `workspace_id` is the primary workspace id and `additional_workpspaces` is an optional kwarg - the document has both (line 153, line 165)